orko-app/src/main/jib/base-docker-config.yml

```diff
@@ -0,0 +1,78 @@
+loopSeconds: ${LOOP_SECONDS:-15}
+scriptSigningKey: ${SCRIPT_SIGNING_KEY:-}
+auth:
+  httpsOnly: ${HTTPS:-false}
+  proxied: ${PROXIED:-false}
+  ipWhitelisting:
+    whitelistExpirySeconds: ${WHITELIST_EXPIRY_SECONDS:-86400}
+    secretKey: ${AUTH_TOKEN:-}
+  jwt:
+    userName: ${SIMPLE_AUTH_USERNAME:-}
```

Comment:
Append secret versions of these so secrets can just be used out-of-the-box without providing a config file